wdl/JoinRawCalls.wdl

```diff
@@ -120,5 +120,6 @@ workflow JoinRawCalls {
   output {
     File joined_raw_calls_vcf = ConcatVcfs.concat_vcf
     File joined_raw_calls_vcf_index = ConcatVcfs.concat_vcf_idx
+    File joined_raw_calls_ploidy_table = CreatePloidyTableFromPed.out
```

Comment:
```suggestion
    File ploidy_table = CreatePloidyTableFromPed.out
```
This ploidy table is not specific to JoinRawCalls, so it makes more sense to just name it ploidy_table. I checked that  there are no other ploidy_table outputs from other workflows so it won't be confusing.

Reply:
Thank you for that comment. The file name was changed as recommended.